inc/spbc-settings.php

```diff
@@ -3394,13 +3424,15 @@ function spbc_list_table__get_args_by_type($table_type)
                 $accordion_default_args,
                 array(
                     'func_data_prepare' => 'spbc_field_scanner__prepare_data__files',
+                    //'func_data_prepare' => 'spbc_field_scanner__approved__data_prepare',
```

Comment:
What this comment for?

Reply:
Fixed https://github.com/CleanTalk/security-malware-firewall/pull/250/commits/e1a534233e13f2ff9276f2cb0b3ab66ff7cc921e